dali/operators/image/color/brightness_contrast.cc

```diff
@@ -24,18 +24,8 @@ using TheKernel = kernels::MultiplyAddCpu<Out, In, 3>;
 }  // namespace
 
 
-DALI_SCHEMA(BrightnessContrast)
-    .DocStr(R"code(Adjusts the brightness and contrast of the images based on the following
-formula::
-
-  out = brightness_shift * output_range + brightness * (grey + contrast * (in - grey))
-
-Where:
-
--	The output_range is 1 for float outputs or the maximum positive value for integral types.
--	Grey denotes the value of 0.5 for ``float``, 128 for ``uint8``, and 16384 for ``int16``, and so on.
-
-This operator can also change the type of data.)code")
+DALI_SCHEMA(Brightness)
+    .DocStr(R"code(Changes the brightness of the image.)code")
```

Comment:
I guess we should keep the full explanation with the formula, adjusted for this case.

Reply:
done